src/backend/backend.cpp

```diff
@@ -54,7 +54,6 @@ namespace big
 		{
 			looped::self_wanted();
 			looped::self_hud();
-			looped::self_dance_mode();
```

Comment:
From Shiny's notes:

> I found two more bugs:
> - dance mode doesn't work, it even breaks your nightclub, you can't enter the property
> - pattern of load_from_datafile is outdated

Reply:
The main reason it was removed isn’t solely because it wasn’t working, but because these features can now be converted into a Lua script, as we have exposed script patches and functions. The same applies to LSC. They don't need to be in the base YimMenu.

Reply:
This feature is used very infrequently and convert it to Lua should have little impact. Please try not to convert frequently used features that are already located within YimMenu to Lua, not all users follow the PR and Lua repositories and they don't know where to find these features. These features are already localized in other languages, and converting them to Lua will make multilingual support more difficult.

Reply:
Can't we add a little window to download these Lua's via the menu itself?

Reply:
> Also at some point we should look into making a sort in-menu Lua workshop to install scripts with.

I can do that right now

Reply:
> > Also at some point we should look into making a sort in-menu Lua workshop to install scripts with.
> 
> I can do that right now

That would be amazing if you could get that implemented.